_includes/faq.scss

```
@@ -44,301 +44,103 @@
-    font-weight: 600;
-    color: $color-RO_donkerblauw;
-    font-size: 18px;
-    margin-top: 27px;
-    margin-bottom: 6px;
-    line-height: 20px;
-    
-    .tabbable-js-interactive & {
-        @media (min-width: 740px) {
-            position: absolute;
-            left: -9999px;
-        }
-    }
+    li:first-child .fag-item{
+        border-top: none;
+    } 
 }
 
-
-
-
-
-
-
-
-
-
-.faq details {
+.fag-item {
+    display: block;
     position: relative;
-    border: 1px solid #154273 /* RO blauw */;
-    border-radius: 5px;
+    color: #154273;
+    border-top: 1px solid rgba(21, 66, 115, 0.5);
+    padding: 0.8em 0.5em;
+    color: #154273;
+    font-size: 21px;
+    line-height: 22px;
     width: 100%;
-    margin-bottom: 10px;
     overflow: hidden;
 
     &:hover {
         background-color: rgba(0, 39, 131, 0.07) /* RO donker link2 */;
-        border-color: $color-RO_donkerblauw--interaction;
+        // border-color: $color-RO_donkerblauw--interaction;
         transition: var(--transition-in);
     }
 
-    &:last-child {
-        margin-bottom: 0;
-    }
-}
-
-.faq details:first-child {
-    margin-top: 0;
-}
-
-.faq details[open],
-.faq details[open]:hover {
-    background: #FFFFFF;
-    border-color: #FFFFFF;
-}
-
-.faq details summary:focus {
-    outline: 0; // See summary::after
-}
-
-.faq details summary::after {
-    content: '';
-    position: absolute;
-    top: 6px;
-    right: 5px;
-    height: 40px;
-    width: 40px;
-    opacity: 0.62;
-    border-radius: 2px;
-    outline-offset: 8px;
-    outline: 2px solid transparent;
-    background: url('../img/plus.svg') center center no-repeat;
-    transition: var(--transition-out);
-}
-
-.faq details summary::before {
-    display: none; // Created by polyfill for IE11
-}
-
-.faq details[open] summary::after {
-    opacity: 1;
-    background-image: url('../img/minus.svg');
-}
-
-.faq details summary:hover::after {
-    opacity: 1;
-    background-color: $color-focus-background;
-}
-
-.faq details summary:focus::after {
-    opacity: 1;
-    background-color: $color-focus-background;
-    transition: var(--transition-in);
-    outline: 2px solid $color-focus;
-    outline-offset: -2px;
-}
-
-.faq details:hover summary::after,
-.faq details:focus summary::after {
-    opacity: 1;
-}
-
-.faq details > summary::-webkit-details-marker {
-    display: none;
-}
-
-.faq details:hover summary,
-.faq details summary:focus {
-    text-decoration: underline;
-    border-color: $color-RO_donkerblauw--interaction;
-}
-.faq details[open]:hover summary,
-.faq details[open] summary:focus {
-    text-decoration: none;
-}
-
-.faq details summary {
-    padding: 15px 0 16px 0;
-    list-style-type: none;
-    cursor: pointer;
-    color: #154273 /* RO blauw */;
-
-    &:hover {
-        user-select: none; // This is to prevent a double click selecting all text.
-        // Normal text selection will be minimally   affected in the text flow.
+    &:focus {
+        outline-offset: 0px;
```

De focus is niet goed zichtbaar als je met je toetsenbord op deze link staat. 

Als we het naar binnen trekken is ie wel goed zichtbaar, misschien is dat een idee:

```suggestion
    &:focus {
        outline-offset: -2px;
```